compiler/onecc-docker/onecc-docker.py

```diff
@@ -0,0 +1,64 @@
+import sys
+import subprocess
+from distutils.version import StrictVersion
+import json
+
+def _is_image(version) : 
+    cmd = ['docker', 'images', f'onecc:{version}']
+    p = subprocess.Popen(cmd,  stdout=subprocess.PIPE, stderr=subprocess.STDOUT) 
+    out, err = p.communicate()
+    dic = {}
+
+    decoded_out = [i.split() for i in out.decode().split("\n") if len(i)]
+
+    if len(decoded_out) <2 :
+        return False
+
+    dic[decoded_out[0][0]] = decoded_out[1][0]
+    dic[decoded_out[0][1]] = decoded_out[1][1]
+
+    tag_name = cmd[2].split(':')
+
+    if dic['REPOSITORY'] == tag_name[0] and dic['TAG'] == tag_name[1]  :
+        return True 
+    else :
+        return False
+
+
+def main():
+    installed_onecc_version = sys.argv[1]
+    versions_str = subprocess.check_output("curl -s https://api.github.com/repos/Samsung/ONE/tags", shell=True ,encoding='utf-8')
+    versions_json = json.loads(versions_str)
+    versions = []
+    for i in versions_json:
+        versions.append(i["name"])
+
+    versions.sort(key=StrictVersion)
+    chosen_version = ""
+    while 1:
+        for i in range(18, len(versions)):
+            print(versions[i])
+        chosen_version = input("choose version: ")
```

Comment:
I think we shouldn't force users to select the version every time. It means that a default version will do the trick. If users don't specify the version, we can assume they want to run `one-compiler` with latest version.